lib/data/demos.dart

```diff
@@ -83,8 +118,43 @@ class GalleryDemoConfiguration {
   final CodeDisplayer code;
 }
 
-List<GalleryDemo> materialDemos(BuildContext context) {
-  final localizations = GalleryLocalizations.of(context);
+List<GalleryDemo> allGalleryDemos(GalleryLocalizations localizations) =>
+    studies(localizations).values.toList() +
+    materialDemos(localizations) +
+    cupertinoDemos(localizations) +
+    otherDemos(localizations);
+
+Map<Symbol, GalleryDemo> studies(GalleryLocalizations localizations) {
+  return <Symbol, GalleryDemo>{
+    #shrine: GalleryDemo(
```

Comment:
What is `Symbol`?

Reply:
I thought it was the Dart type for symbols but it's to be used for methods and members instead (https://api.flutter.dev/flutter/dart-core/Symbol/Symbol.html). I'll replace with strings